Sources/Utility/Git.swift

```diff
@@ -47,11 +48,9 @@ public class Git {
         public var sha: String! {
             return try? Git.runPopen([Git.tool, "-C", path, "rev-parse", "--verify", "HEAD"]).chomp()
         }
-        
         public func versionSha(tag: String) throws -> String {
             return try Git.runPopen([Git.tool, "-C", path, "rev-parse", "--verify", "\(tag)"]).chomp()
         }
-        
```

Comment:
Can you discard these diffs? If you're having trouble looking at whitespaces diff you can use a good GUI client which can show the whitespace diff properly. (I sometimes use gitX however its quite old... I am sure very good GUI clients are available)